### DISCUSSION TITLE
UI inside Iframe not getting displayed on trying to load from cypress

### DISCUSSION THREAD

**Discussion by ghost** [cypress-io/cypress#28824] [Questions and Help]
Hi All,

I am trying load Test application using cypress. post the login(which is successful). The UI inside the iframe is not getting displayed. reset of the UI loading without issue. Any idea why UI is not loading.

**On manually loading it looks like below:-**
![image](https://github.com/cypress-io/cypress/assets/158160149/59f444f2-d606-41e0-ab3e-14354a4911c5)

**UI inside the iframe:-**
![image](https://github.com/cypress-io/cypress/assets/158160149/f3b288ed-b3d0-46af-831a-44ff32cd8670)

**Screenshot from Cypress Run**

![image](https://github.com/cypress-io/cypress/assets/158160149/d0a9b78d-29ce-43a0-84ef-3de2b93fd59f)

Code Snippet:
import 'cypress-iframe'

describe('My First Test', () => {
  it('Visits the Kitchen Sink', () => {
    cy.visit('https://test.sonic.bankwest.com.au/HomeLendingPortalContainer')
    cy.get('#BrokerId').type('')
    cy.get('#Password').type('')
    cy.get('.button').click()

     cy.wait(15000)
     cy.reload()

  })
})

**Comment by amargupta22** [comment]
I am also facing same problem. Please help.
